api/status_test.go

```diff
@@ -22,6 +22,11 @@ func TestStatuses(t *testing.T) {
 	defer WebSocketClient.Close()
 	WebSocketClient.Listen()
 
+	time.Sleep(300 * time.Millisecond)
+	if resp := <-WebSocketClient.ResponseChannel; resp.Status != model.STATUS_OK {
```

Comment:
It doesn't need any handling really, this is only here because the unit tests rely on the responses coming in a certain order. In a real world situation you would have a reader listening to the ResponseChannel and handling all the responses you care about, while discarding the ones you don't care about